### PR TITLE
feat(eagleye): split fix2pose

### PR DESCRIPTION
**Pull request by kminoda**
## Description

MUST BE MERGED WITH https://github.com/MapIV/eagleye/pull/312

[INTERNAL JIRA LINK](https://tier4.atlassian.net/browse/RT1-3575)

## Tests performed

See https://github.com/MapIV/eagleye/pull/312

## Effects on system behavior

No effect expected

## Pre-review checklist for the PR author

The PR author **must** check the checkboxes below when creating the PR.

- [x] I've confirmed the [contribution guidelines].
- [x] The PR follows the [pull request guidelines].

## In-review checklist for the PR reviewers

The PR reviewers **must** check the checkboxes below before approval.

- [ ] The PR follows the [pull request guidelines].

## Post-review checklist for the PR author

The PR author **must** check the checkboxes below before merging.

- [ ] There are no open discussions or they are tracked via tickets.

After all checkboxes are checked, anyone who has write access can merge the PR.

[contribution guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/
[pull request guidelines]: https://autowarefoundation.github.io/autoware-documentation/main/contributing/pull-request-guidelines/

### CODE REVIEW

**Review by KYabuuchi — COMMENTED** [review]
Please update `tier4_localization_launch/package.xml` :pray:
```diff
+ <exec_depend>eagleye_geo_pose_converter</exec_depend>
+ <exec_depend>eagleye_geo_pose_fuser</exec_depend>
- <exec_depend>eagleye_fix2pose</exec_depend>
```

**Comment by kminoda** [comment]
@KYabuuchi Thank you! Done https://github.com/autowarefoundation/autoware.universe/pull/5506/commits/1426d872a86381d741e9af53bcc06698023fe0b8

File: launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml [hunk 1]
```diff
@@ -1,30 +1,23 @@
...
+
     <!-- plane rectangular coordinate number -->
     <param name="plane" value="7"/>
     <!-- 1 : plane rectangular coordinate  2 : MGRS -->
     <param name="tf_num" value="2"/>
     <!-- 0 : No convert  1 : ellipsoid -> altitude  2 : altitude -> ellipsoid -->
     <param name="convert_height_num" value="0"/>
-    <!-- 0 : EGM2008-1  1 : Japanese Geoid 2011 Ver2.1 -->
+    <!-- cspell:ignore GSIGE2011 -->
+    <!-- 0 : EGM2008-1  1 : GSIGE2011 Ver2.1 -->
```
Comment (KYabuuchi) [inline launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml:15]: Do you mean `GSIGEO2011`?
[related link](https://www.gsi.go.jp/ENGLISH/Bulletin62_CONTENTS.html#62-2:~:text=Development%20of%20new%20hybrid%20geoid%20model%20for%20Japan%2C%20%22GSIGEO2011%22)

**Review by KYabuuchi — COMMENTED** [review]
I have confirmed that it works. I will approve soon.

File: launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml (hunk 1 above)
Comment (KYabuuchi) [reply launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml:15]: @rsasaki0109  FYI: eagleye has also have an inccorect notation?
https://github.com/MapIV/eagleye/blob/30bae75dace6d9bfed0d67d51d6eec5928932509/eagleye_util/fix2pose/launch/fix2pose.xml#L15

Comment (rsasaki0109) [reply launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml:15]: Thank you for pointing it out. It's a misspelling.

Comment (kminoda) [reply launch/tier4_localization_launch/launch/pose_twist_estimator/eagleye/geo_pose_converter.launch.xml]: Thanks, corrected: https://github.com/autowarefoundation/autoware.universe/pull/5506/commits/d64b8feed171573d63cf9b255028d453aef18d7e

**Review by KYabuuchi — APPROVED** [review]
LGTM

**Pull request merged**